rotkehlchen/db/filtering.py

```diff
@@ -646,6 +647,13 @@ def make(
         if location is not None:
             filter_query.location_filter = DBLocationFilter(and_op=True, location=location)
             filters.append(filter_query.location_filter)
+        if trades_idx_to_ignore is not None:
+            filters.append(DBIgnoreValuesFilter(
+                and_op=True,
+                column='id',
+                # only iteration is performed, so converting to a list is not efficient
+                values=trades_idx_to_ignore,  # type: ignore[arg-type]
```

Comment:
Why not simpy change the type of `DBIgnoreValueFilter(values)` to also allow set?